src/Api/config.js

```diff
@@ -1,3 +1,6 @@
 export default /localhost|staging/.test(window.location.host)
 	? 'https://eagle-bn-backend-staging.herokuapp.com/api/v1/'
 	: 'https://eagle-bn-backend.herokuapp.com/api/v1/';
+export const url = /localhost|staging/.test(window.location.host)
```

Comment:
I suggest to let this file as it was. because this doesn't make any impact(it will only increase change in this PR)
for better, you can change this line to `/localhost|staging/fro-/test(...)` to capture review apps too.

Reply:
The second line does not contain api/v1 I used it to connect to socket client

Reply:
sorry for  my incovenience

Reply:
Nothing to be sorry my bro. Thanks!